src/resources/gallery.ts

```diff
@@ -0,0 +1,34 @@
+import BaseResource from './base';
+
+export interface Thumbnail {
+  readonly url: string;
+  readonly width: number;
+  readonly height: number;
+}
+
+export default class GalleryResource extends BaseResource {
+  static urlRoot =
+    'https://ymun-app.caseychow.org/_/items/gallery_photos?fields=*.*&sort=-modified_on';
```

Comment:
Can’t add params to the URL root because then it doesn’t know how to get individual items. I think there’s another way but I mostly gave up and just sequencer two useResource calls.